### DISCUSSION TITLE
Is there anyway to make code actions available in a select menu like coc-nvim?

### DISCUSSION THREAD

**Discussion by npearson72** [hrsh7th/nvim-cmp#1369] [Q&A] [answered]
Here's how they do it:

https://user-images.githubusercontent.com/1323414/209429323-48ca2df2-7c32-452d-9202-3746738c30e6.mp4

Is there a config option for this, or would this be a feature request?

**Comment by gegoune** [comment, marked as answer]
Cmp doesn't handle code actions at all, I believe. You can use [telescope ui select](https://github.com/nvim-telescope/telescope-ui-select.nvim) extension to achieve that look/behaviour.
